cmd/pixelizer/image.go

```diff
@@ -9,17 +10,19 @@ import (
 	"image/png"
 	"os"
 	"strconv"
+
+	log "github.com/sirupsen/logrus"
 )
 
 const GifDelayEachFrame = 20
 
-func launchPixellisator(f *os.File, min int, count int, increase int) {
+func launchPixellisator(f *os.File, min int, count int, increase int) error {
 	logIfVerbose(DEBUG, "launchPixellisator[ %v / %v / %v / %v ]", f.Name(), min, count, increase)
-	img, _, errImageDecode := image.Decode(f)
-	logIfExists(errImageDecode)
```

Comment:
Why did you remove this method ? It was quite convenient to lighten the code.

Reply:
That is just a preferred way of error handling in the go community. More about semantic, not an issue